.github/workflows/integration_test.yaml

```diff
@@ -244,7 +244,8 @@ jobs:
         working-directory: ${{ inputs.working-directory }}
         run: |
           sudo snap install charmcraft --classic --channel latest/stable
-          charmcraft pack -v
+          python -m pip install tox
+          tox -e pack -- -v | charmcraft pack -v
```

Comment:
should be "||", no?

Reply:
I think you're right, I have fixed that. I wonder why it worked anyway, I think `|` is bitwise or which probably is fine as well in this case